src/plugins/cosmos/components/modals/Staked/views/Staked.tsx

```diff
@@ -0,0 +1,74 @@
+import { Box, Flex } from '@chakra-ui/layout'
+import { Button, ModalCloseButton } from '@chakra-ui/react'
+import { Asset } from '@shapeshiftoss/types'
+import { AnimatePresence } from 'framer-motion'
+import { RewardsRow } from 'plugins/cosmos/components/RewardsRow/RewardsRow'
+import { StakedHeader } from 'plugins/cosmos/components/StakedHeader/StakedHeader'
+import { StakedRow } from 'plugins/cosmos/components/StakedRow/StakedRow'
+import { StakingButtons } from 'plugins/cosmos/components/StakingButtons/StakingButtons'
+import { UnbondingRow } from 'plugins/cosmos/components/UnbondingRow/UnbondingRow'
+import { Text } from 'components/Text'
+import { bnOrZero } from 'lib/bignumber/bignumber'
+
+type StakedProps = {
+  assetId: string
+}
+
+// TODO: Wire up the whole component with staked data
+export const Staked = ({ assetId }: StakedProps) => {
+  // TODO: wire me up, parentheses are nice but let's get asset name from selectAssetNameById instead of this
+  const asset = (_ => ({
+    name: 'Osmo',
+    symbol: 'OSMO'
+  }))(assetId) as Asset
+  return (
+    <AnimatePresence exitBeforeEnter initial={false}>
+      <Box pt='38px' pb='70px' px='34px'>
+        <ModalCloseButton borderRadius='full' />
+        <Flex
+          direction='column'
+          maxWidth='595px'
+          minHeight='380px'
+          alignItems='center'
+          justifyContent='space-between'
+        >
+          <StakedHeader assetName={asset.name} width='100%' mb='35px' />
+          <StakedRow
+            width='100%'
+            mb='10px'
+            assetSymbol={asset.symbol}
+            fiatRate={bnOrZero('8.47')}
+            cryptoStakedAmount={bnOrZero('708.00')}
+            apr={bnOrZero('1.25')}
+          />
+          <StakingButtons width='100%' />
+          <Box width='100%' mt='20px'>
+            {
+              /* TODO: use real unbonds data */
+              new Array(2).fill(undefined).map((_, i) => (
```

Comment:
`useMemo` here as a map will cause this to rerender every time

Reply:
Yeah right now this is unstable, but when we will start to use real unbonds data, it will be referentially stable because of reselect's memoization, so IMO there's no point to add a `useMemo` call just to remove it later